chapters/lighting/sections/environment/environment.tex

```diff
@@ -0,0 +1,66 @@
+\section{Environment}
+Even though the game can be played in non-Euclidean spaces which makes it inherently unrealistic, we decided to add some elements that would make the scenes portrayed in the game feel familiar.
+One such element is the Earth-like terrain, described in detail in the previous sections.
+Another property of the real world that we wanted to capture in the game was the daytime cycle.
+In the game, the full cycle is 10 minutes long, with 5 minutes long daytime and 5 minutes long nighttime.
+We also added transitions between day and night to give the Earth-like experience of sunrise and sunset.
+The scene during various times of the day is shown in \autoref{fig:cycle}.
+
+\begin{figure*}[h]
+    \centering
+    \begin{subfigure}[b]{0.475\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{chapters/lighting/sections/environment/resources/day.png}
+        \caption[]%
+        {{\small Day}}
+        \label{fig:cycle-day}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.475\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{chapters/lighting/sections/environment/resources/night.png}
+        \caption[]%
+        {{\small Night}}
+        \label{fig:cycle-night}
+    \end{subfigure}
+    \vskip\baselineskip
+    \begin{subfigure}[b]{0.475\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{chapters/lighting/sections/environment/resources/sunrise.png}
+        \caption[]%
+        {{\small Sunrise}}
+        \label{fig:cycle-sunrise}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.475\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{chapters/lighting/sections/environment/resources/sunset.png}
+        \caption[]%
+        {{\small Sunset}}
+        \label{fig:cycle-sunset}
+    \end{subfigure}
+    \caption[]
+    {\small Day night cycle in the game}
+    \label{fig:cycle}
+\end{figure*}
+
+The implementation of the day-night cycle relies on two components: directional lighting (described in \autoref{subsec:directional-lighting}) which corresponds to the light coming from the sun and a \textit{skybox} representing the sky.
+Conceptually, a skybox is a cube made out of 6 images, one per each side, that encompasses the scene thus creating an illusion that the world is much bigger than it is in reality.
+A skybox can be implemented in OpenGL using a special type of texture, a \textit{cubemap}, i.e. a texture that contains 6 individual 2D textures.
+In the game, we're using images of the night sky obtained from an HDR file \url{https://www.reddit.com/r/blender/comments/3ebzwz/free_space_hdrs_1/} using an online utility program \url{https://matheowis.github.io/HDRI-to-CubeMap/}.
```

Comment:
Maybe add a \todo here to ask promotor about the use of this just to be sure

Reply:
We can but we're pretty much screwed if we can't use it